ansible/edge.yml

```diff
@@ -8,3 +8,4 @@
   roles:
   - nginx
   - cli
+  - cli-install
```

Comment:
You can do it here, but you need to run under hosts: ansible; that's why it's installing the 'x86' version instead of the 'mac' version on your docker-machine.

Reply:
It may also be necessary to look at how the `cli-install` scripts determine the nginx host to download from; I think I may have fudged that late at night.